database/migrations/2017_04_27_141453_create_submissions_table.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSubmissionsTable extends Migration
```

Comment:
Submission is like a scratchpad that has states "in progress", "awating review", "accepted", "rejected", "paid", "archived". supports
- users need to be able to construct a list to be dispatched. 
- user need to pickup where they left off. 
- users need to know whether thier submission has been paid or rejected (for some reason)
- the app needs to support progressive enhancement , so while the endpoint for submission might accept a fully realised client produced Submission datastructure it doesn't *have* to.

(though, there is a danger i'm overthinking this.)